python/lib/dependabot/python/file_updater/pipfile_file_updater.rb

```diff
@@ -233,20 +233,20 @@ def post_process_lockfile(updated_lockfile_content)
 
         def generate_updated_requirements_files
           req_content = run_pipenv_command(
-            "pyenv exec pipenv lock -r"
+            "pyenv exec pipenv requirements"
           )
           File.write("req.txt", req_content)
 
           dev_req_content = run_pipenv_command(
-            "pyenv exec pipenv lock -r -d"
+            "pyenv exec pipenv requirements --dev"
           )
           File.write("dev-req.txt", dev_req_content)
         end
 
         def run_command(command, env: {})
           start = Time.now
           command = SharedHelpers.escape_command(command)
-          stdout, process = Open3.capture2e(env, command)
+          stdout, _, process = Open3.capture3(env, command)
```

Comment:
This was extracted out separately into:
* https://github.com/dependabot/dependabot-core/pull/6103